### PR TITLE
Fix 1.4

### PR DESCRIPTION
**Pull request by skeleton**
| Q                 | A
| ----------------- | ---
| Specs             | N
| Behats            | Fix behats
| Blue CI           | ?
| Changelog updated | Y
| Review and 2 GTM  | -

### CODE REVIEW

File: features/deprecated-behat/import/browse_imports.feature [hunk 1]
```diff
@@ -4,6 +4,7 @@ Feature: Browse imports
   As a product manager
   I need to be able to view a list of them
 
+  @ce
```
Comment (phaseinducer) [inline features/deprecated-behat/import/browse_imports.feature]: Does it work ? Why not playing this scenario in EE ?

**Comment by jjanvier** [comment]
gtm

File: src/Pim/Bundle/ReferenceDataBundle/DataGrid/Datasource/ResultRecord/MongoDbOdm/Product/ReferenceDataTransformer.php [hunk 2]
```diff
@@ -25,10 +25,11 @@ class ReferenceDataTransformer
     public function transform(array $result, array $attribute, $locale, $scope)
     {
         $attributeCode = $attribute['code'];
-        $normalizedData = $result['normalizedData'];
-        $fromNormData = ['pim_reference_data_simpleselect', 'pim_reference_data_multiselect'];
+        $properties = $attribute['properties'];
+
+        if (isset($properties['reference_data_name']) && '' !== $properties['reference_data_name']) {
```
Comment (grena) [inline src/Pim/Bundle/ReferenceDataBundle/DataGrid/Datasource/ResultRecord/MongoDbOdm/Product/ReferenceDataTransformer.php]: Simple question, what are we checking here?

**Comment by grena** [comment]
Just a question, but GTM anyway

**Comment by phaseinducer** [comment]
specs are broken by your fix
GTM otherwise

File: features/deprecated-behat/import/browse_imports.feature (hunk 1 above)
Comment (skeleton) [inline features/deprecated-behat/import/browse_imports.feature]: Scenario fails on `Then the grid should contain 7 elements`
We have 7 jobs in CE, but 8 in EE.
And yes, it works

File: src/Pim/Bundle/ReferenceDataBundle/DataGrid/Datasource/ResultRecord/MongoDbOdm/Product/ReferenceDataTransformer.php (hunk 2 above)
Comment (skeleton) [inline src/Pim/Bundle/ReferenceDataBundle/DataGrid/Datasource/ResultRecord/MongoDbOdm/Product/ReferenceDataTransformer.php]: We check if current attribute is a reference data. 
Before we had static attribute type. But it doesn't work on mongo EE with assets.

File: features/deprecated-behat/import/browse_imports.feature (hunk 1 above)
Comment (phaseinducer) [inline features/deprecated-behat/import/browse_imports.feature]: Wouldn't it be better to remove the assertion "the grid should contain 7 elements" and keep this in EE ?

**Comment by ClementGautier** [comment]
GTM after travis fixed

File: features/deprecated-behat/import/browse_imports.feature (hunk 1 above)
Comment (skeleton) [inline features/deprecated-behat/import/browse_imports.feature]: nope, this scenario : 
- count items in list
- check what jobs are return with filters

If I remove all lines which will break CI, this scenario has no sense

File: src/Pim/Bundle/ReferenceDataBundle/DataGrid/Datasource/ResultRecord/MongoDbOdm/Product/ReferenceDataTransformer.php (hunk 2 above)
Comment (grena) [inline src/Pim/Bundle/ReferenceDataBundle/DataGrid/Datasource/ResultRecord/MongoDbOdm/Product/ReferenceDataTransformer.php]: Thanks :)

**Pull request merged**